example/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "example",
+  "version": "0.0.1",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "npm run compile && node index.js",
+    "compile": "faucet --no-fingerprint --compact"
+  },
+  "author": "Falk Hoppe <falkhoppe81@gmail.com>",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "complate-express": "file:..",
+    "express": "^4.15.3"
+  },
+  "devDependencies": {
+    "complate-jsx": "^0.7.1"
```

Comment:
This is not a problem of this package — especially because it is just for the example part — but I recommend a layered build, in which dev-dependencies are installed, used and removed before released.